infra/test/integration/simple_example/simple_example_test.go

```diff
@@ -32,7 +32,7 @@ func TestSimpleExample(t *testing.T) {
 	example := tft.NewTFBlueprintTest(t)
 
 	example.DefineVerify(func(assert *assert.Assertions) {
-
+		example.DefaultVerify(assert)
```

Comment:
This `DefaultVerify` method would help us find the different between the Terraform code/definition and the actual state of our Google Cloud resources _after_ `terraform apply`.